samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright 2018, Google, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function localizeObjects(file) {
+  // [START vision_localize_objects]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+  const fs = require('fs');
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const fileName = 'Local image file, e.g. /path/to/image.png';
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+  const request = {
+    image: {content: fs.readFileSync(file)},
+  };
+
+  client
+    .objectLocalization(request)
+    .then(results => {
+      console.log(JSON.stringify(results, null, '\t'));
```

Comment:
The response processing is missing, this simply dumps the response object to screen

What is the canonical sample? If you wouldn't mind, please update the description of this Pull Request to include the canonical samples this PR is for, right now the description shows an incomplete template which was not filled out

Here is an example of a standard PR description: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/574